flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/utils/JsonPlanTestBase.java

```diff
@@ -72,7 +73,9 @@ public void after() {
     protected TableResult compileSqlAndExecutePlan(String sql) {
         CompiledPlan compiledPlan = tableEnv.compilePlanSql(sql);
         checkTransformationUids(compiledPlan);
-        return compiledPlan.execute();
+        String jsonPlan = compiledPlan.asJsonString();
```

Comment:
add some comment about the magic here is trying to validate the ser/de of plan json works well.